modules/local/colabfold_batch.nf

```diff
@@ -3,7 +3,7 @@ process COLABFOLD_BATCH {
     label 'process_medium'
 
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'docker://quay.io/nf-core/proteinfold_colabfold:1.1.0' :
```

Comment:
I think we need the `quay.io` in the URL, don't we?

Reply:
I mean the container is not in dockerhub anymore, should it be?

Reply:
I had the same question!

Reply:
It's not necessary, since the default docker.registry is quay.io